packages/selected-network-controller/tests/SelectedNetworkController.test.ts

```diff
@@ -781,92 +846,20 @@ describe('SelectedNetworkController', () => {
     });
   });
 
-  describe('Constructor checks for domains in permissions', () => {
-    describe('when useRequestQueuePreference is true', () => {
-      it('should set networkClientId for domains not already in state', async () => {
```

Comment:
isn't this test still valid?

Reply:
these were moved into the existing `constructor` scenario. See https://github.com/MetaMask/core/pull/4409/files#diff-a9e0027991e57fd3175c02854cb99119a961b43dedc6758e0a117ebe6321e428R230